ASCII.c

```diff
@@ -0,0 +1,10 @@
+#include<stdio.h>
+
+int main(){
+    char c;
+    printf("Nhap 1 ky tu bat ky: ");
+    scanf("%c",&c);
```

Comment:
@spm1102 
`#reply_required`
`#for_information`

After `,` should have **a space**

```c
    scanf("%c", &c);
```

Reply:
@Dark-GreenCat U are right! I'll definitely fix that.